props/saber_BC_buttons.h

```diff
@@ -2117,6 +2120,17 @@ void DoSavedTwist() {
 
   RefPtr<BufferedWavPlayer> wav_player;
 
+  bool Event(enum BUTTON button, EVENT event) override {
+#ifdef BC_BUTTON_CLICKER
+    // Play clicker only when saber is OFF and a button is PRESSED
+    if (!IsOn() && event == EVENT_PRESSED &&
+        (button == BUTTON_POWER || button == BUTTON_AUX || button == BUTTON_AUX2)) {
```

Comment:
I think this is fine, but I'm curious, why not just do it for all buttons?

Reply:
Ah I see. Why specify?
Done.